components/widgets/inputs/users_emails_input.jsx

```diff
@@ -258,32 +264,54 @@ export default class UsersEmailsInput extends React.PureComponent {
             return {label: v, value: v};
         });
         return (
-            <AsyncSelect
-                ref={this.selectRef}
-                styles={this.customStyles}
-                onChange={this.onChange}
-                loadOptions={this.optionsLoader}
-                isValidNewOption={this.showAddEmail}
-                isMulti={true}
-                isClearable={false}
-                className={classNames('UsersEmailsInput', {empty: this.props.inputValue === ''})}
-                classNamePrefix='users-emails-input'
-                placeholder={this.props.placeholder}
-                components={this.components}
-                getOptionValue={this.getOptionValue}
-                formatOptionLabel={this.formatOptionLabel}
-                defaultOptions={false}
-                defaultMenuIsOpen={false}
-                openMenuOnClick={false}
-                loadingMessage={this.loadingMessage}
-                onInputChange={this.handleInputChange}
-                inputValue={this.props.inputValue}
-                openMenuOnFocus={true}
-                onFocus={this.onFocus}
-                tabSelectsValue={true}
-                value={values}
-                aria-label={this.props.ariaLabel}
-            />
+            <div>
```

Comment:
You can avoid adding the `div` here and instead use `React.Fragment` so that it renders the block without an unnecessary `div`. There's a shorthand you can use here that I've suggested below:
```suggestion
            <>
```